nodejs/week1/avg.js

```diff
@@ -0,0 +1,15 @@
+function calculateAverageFromArgs() {
+  const args = process.argv.slice(2);
+  const numbers = args.map(Number);
+
+  if (numbers.length === 0 || numbers.some(isNaN)) {
+    console.log("Please provide a list of numbers separated by spaces.");
+  } else {
+    const sum = numbers.reduce((x, y) => x + y, 0);
+    const average = sum / numbers.length;
+    console.log(`The average of ${numbers.join(", ")} is ${average}`);
+  }
+}
+
+// Call the function to execute the code
+calculateAverageFromArgs();
```

Comment:
Nicely done! try to make your function independent of the args. meaning your function should take input parameters. In this way you separate function logic (the calculation of the average), from the data (the input parameters)

Reply:
I tried to make function independent of the arguments now. Thank you for the suggestion!!